sql/core/src/test/scala/org/apache/spark/sql/JoinSuite.scala

```diff
@@ -30,6 +30,15 @@ import org.apache.spark.TestUtils.{assertNotSpilled, assertSpilled}
 class JoinSuite extends QueryTest with SharedSQLContext {
   import testImplicits._
 
+  override def afterEach(): Unit = {
+    try {
+      // Clear the cache table for test cases
+      spark.sharedState.cacheManager.clearCache()
```

Comment:
Does [this](https://github.com/apache/spark/blob/master/sql/core/src/test/scala/org/apache/spark/sql/test/SharedSQLContext.scala#L91) work for `JoinSuite`, too?